dev_guide/console.adoc

```diff
@@ -69,34 +69,3 @@ image:console_filter_step3.png["Console Filter Step 3"]
 
 image:console_filter_active.png["Console Filter Active"]
 ====
-
-
-== Creating from Template
```

Comment:
Ah ok. @bparees I see what you mean now. Including this doc as a whole might be something I can talk to @CowboysFan  about. As for now, I've added a small section to the console doc about creating from templates, linking to here. It's really a placeholder; once we settle on what else should be here it might work better as a list or table or something.